### DISCUSSION TITLE
Standalone windows

### DISCUSSION THREAD

**Discussion by laguinch** [photobooth-app/photobooth-app#128] [Q&A] [answered]
Hy, when I chose the standalone ip address, after all new picture the web browser refresh on the picture take windows (http://........:8000) and not on the standalone page. Is that normal ? Thx

**Comment by mgineer85** [comment]
Ah, could be a bug, need to check.

**Reply by mgineer85** [reply, marked as answer]
This one has been fixed some releases ago already.
